public/js/lib/devtools/client/shared/shim/Services.js

```diff
@@ -432,35 +414,32 @@ PrefBranch.prototype = {
       this._findOrCreatePref(event.key, userValue, hasUserValue, defaultValue);
     } else {
       let thePref = this._findPref(event.key);
-      thePref.storageUpdated(type, userValue, hasUserValue, defaultValue);
+      thePref._storageUpdated(type, userValue, hasUserValue, defaultValue);
     }
   },
 
   /**
    * Helper function to initialize the root PrefBranch.
    */
   _initializeRoot: function () {
-    try {
-      if (localStorage.length === 0) {
-        // FIXME - this is where we'll load devtools.js to install the
-        // default prefs.
-      }
-    } catch(e) {
-      // Couldn't access localStorage; bail. This happens in the
-      // Firefox panel because Chrome-privileged code can't access it.
-      return;
+    if (localStorage.length === 0 && Services._defaultPrefsEnabled) {
+      /* eslint-disable no-eval */
+      let devtools = require("raw!prefs!devtools/client/preferences/devtools");
+      eval(devtools);
+      let all = require("raw!prefs!modules/libpref/init/all");
+      eval(all);
+      /* eslint-enable no-eval */
```

Comment:
This is the code that's failing. We'll need to see where the prefs loader is defined